index.html

```diff
@@ -176,50 +241,80 @@ <h3 id="matters-considered-by-the-placement-rules">Matters considered by the pla
 <ol>
   <li>Ruby is used to display the reading or the meaning of the base characters.
     Therefore, the number one priority here is to avoid misreadings.
-    <aside class=note title="Ruby as notes">
-      Notes are sometimes placed between lines
-      similarly to how ruby is laid out
-      (inter-linear notes).
-      The processing of this arrangement is not covered in this document.
+    <span class="checkme">
+    Specifically, the ruby string which protrudes from the base character 
+    string is not allowed to be laid over the characters preceding or 
+    following, nevertheless preceding or following characters are Kanji or 
+    Kana characters. 
```

Comment:
> characters preceding or following, nevertheless preceding or following characters are Kanji or Kana characters.

What about changing it to something like this:

> preceding or following character, whether it is a Kanji or a Kana character.

Reply:
thank you for suggestion. yes, that is smarter and let me change to that.